docker-bake.hcl

```diff
@@ -21,15 +47,24 @@ target "binaries" {
 }
 
 target "all-arch" {
-  platforms = ["linux/amd64", "linux/arm64", "linux/arm", "linux/ppc64le", "linux/s390x", "linux/riscv64", "linux/386", "linux/mips64le"]
+  platforms = [
+    "linux/amd64",
+    "linux/arm64",
+    "linux/arm",
+    "linux/ppc64le",
+    "linux/s390x",
+    "linux/riscv64",
+    "linux/386",
+    "linux/mips64le"
+  ]
 }
 
 target "mainline" {
   args = {
-    QEMU_REPO = "https://github.com/qemu/qemu"
-    QEMU_VERSION = "v5.2.0"
+    QEMU_REPO = trimspace(getdef("${QEMU_REPO}", "https://github.com/qemu/qemu"))
```

Comment:
This looks super hacky. I guess it is because of the shared var that is updated in build master. But without it I would prefer `--set` then.

Reply:
> This looks super hacky. I guess it is because of the shared var that is updated in build master.

Yes I would have preferred to use shared var available on current master branch of buildx.

> But without it I would prefer `--set` then.

Yes I will make a change to use `--set`. That's way better.